deploy_solution.sh

```diff
@@ -82,6 +82,8 @@ region="$(echo $DEPLOYMENT_DESCRIPTION | jq -r '.terraformBlueprint.inputValues.
 zone="$(echo $DEPLOYMENT_DESCRIPTION | jq -r '.terraformBlueprint.inputValues.zone.inputValue')"
 project_id = "${PROJECT_ID}"
 deployment_name = "${DEPLOYMENT_NAME}"
+api_image="gcr.io/${PROJECT_ID}/three-tier-app-be:${IMAGE_TAG}"
```

Comment:
Don't see these variables being used in the script. Why do we need to define them here?